chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -808,6 +858,8 @@ void BraveRenderViewContextMenu::InitMenu() {
         index.value() + 1, IDC_CONTENT_CONTEXT_OPENLINK_SPLIT_VIEW,
         IDS_CONTENT_CONTEXT_SPLIT_VIEW);
   }
+
+  BuildBraveScreenshotsMenu();
```

Comment:
nit: maybe this should be `MaybeBuildBraveScreenshotsMenu` as it'll only build the feature if its enabled